src/test/java/bio/terra/service/snapshotbuilder/utils/AggregateBQQueryResultsUtilsTest.java

```diff
@@ -21,4 +21,14 @@ void rollupCountsReturnsListOfInt() {
         AggregateBQQueryResultsUtils.toCount(values),
         equalTo(5));
   }
+
+  @Test
+  void domainIdReturnsListOfString() {
+    FieldValueList row =
+        FieldValueList.of(List.of(FieldValue.of(FieldValue.Attribute.PRIMITIVE, "domain_id")));
+    assertThat(
+        "domainId converts table result to list of String",
```

Comment:
The test name and the assert description appear to be out of date, as it returns a single string not a list.